### ISSUE TITLE
Offer API method to access the IPv6 address of a node. 

### ISSUE THREAD

**Issue by tnull** [mininet/mininet#664]
I know mininet has some problems around IPv6 support, however at my setup it seems to work fine and my hosts automatically configure their IPv6 addresses.
But I fail to extract these addresses via the python API, so they are useless to me.

An equivalent of the `node.IP()` would be highly appreciated here.
Regards!

**Comment by lantz** [comment]
Agreed - although I personally dislike IPv6, we really should come up with a comprehensive way of supporting IPv6 addresses in all of the methods that use IP addresses.

**Comment by blackwiz4rd** [comment]
I also find the necessity to use IPv6 in a simulation environment. Thanks for your tool!

**Comment by apptimise** [comment]
Any updates on this? Is it possible to use/set IPv6 in mininet yet?
